tests/E2E/Cli/CheckCommandTest.php

```diff
@@ -41,6 +41,18 @@ public function test_app_returns_error_with_multiple_violations(): void
         $this->assertCheckHasErrors($cmdTester, $expectedErrors);
     }
 
+    public function test_app_returns_single_error_because_there_is_stop_on_failure_param(): void
+    {
+        $cmdTester = $this->runCheck(__DIR__.'/../_fixtures/configMvc.php', true);
+
+        $expectedErrors = 'ERRORS!
+
+App\Controller\Foo violates rules
+  should implement ContainerAwareInterface because all controllers should be container aware';
+
+        $this->assertCheckHasErrors($cmdTester, $expectedErrors);
```

Comment:
I am trying to make a different implementation, and I discovered that this test doesn't check that the other errors are not shown (the `assertCheckErrors` check with a `contains` not with an `equals`). Just to remember it in case we will decide to merge this PR.